tests/components/zwave_mqtt/common.py

```diff
@@ -0,0 +1,57 @@
+"""Helpers for tests."""
+import json
+
+from homeassistant import config_entries
+from homeassistant.components.zwave_mqtt.const import DOMAIN
+
+from tests.async_mock import Mock, patch
+from tests.common import MockConfigEntry
+
+
+async def setup_zwave(hass, entry=None, fixture=None):
+    """Set up Z-Wave and load a dump."""
+    hass.config.components.add("mqtt")
+
+    if entry is None:
+        entry = MockConfigEntry(
+            domain=DOMAIN,
+            title="Z-Wave",
+            connection_class=config_entries.CONN_CLASS_LOCAL_PUSH,
+        )
+
+        entry.add_to_hass(hass)
+
+    with patch("homeassistant.components.mqtt.async_subscribe") as mock_subscribe:
+        mock_subscribe.return_value = Mock()
+        assert await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+
+    assert "zwave_mqtt" in hass.config.components
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5460460)